docs/docker/01-docker-images.md

```diff
@@ -37,13 +37,38 @@ Due to that:
 
 #### Limited IL2CPP support
 
-Currently images are only available with Ubuntu as the base operating system.
+Currently images are only available with Ubuntu or Windows as the base operating system.
 
 Some platforms require their respective operating systems in order to generate IL2CPP builds.
 
 That's why:
 
-- IL2CPP for Windows is not supported
 - IL2CPP for MacOS is not supported
 
-We are looking to include both Windows and MacOS as base images "in the future", which is mostly dependent on contributions from the community.
+We are looking to include MacOS as a base image "in the future", which is mostly dependent on contributions from the community.
+
+If you are looking to generate IL2CPP builds for MacOS, you can do so via [Github Actions](/docs/github/getting-started#il2cpp-example) without a docker container.
+
+#### Concurrent Builds on Windows and MacOS
+
+Windows and MacOS will consume a license seat for each concurrent build instance. This is in contrast to Linux based builds which consume the same seat allowing
+for unlimited simultaneous instances on the same license seat. This is not an issue for free licenses, but for paid licenses, you will need to be mindful of
+starting too many parallel jobs as activation will fail. Below are some examples of number of consumed seats for a build:
+
+You are building for 1 target on a Linux based platform, 1 target for a Windows based platform, and 1 target for a MacOS based platform:
+
+- Number of seats consumed for the Linux build instances: 1
+- Number of seats consumed for the Windows build instances: 1
+- Number of seats consumed for the MacOS build instances: 1
+
+Total concurrently consumed seats: **3**
+
+You are building for 3 targets on a Linux based platform, 3 targets for a Windows based platform, and 1 target for a MacOS based platform:
+
+- Number of seats consumed for the Linux build instances: 1
+- Number of seats consumed for the Windows build instances: 3
```

Comment:
In my project, I'm building for 4 targets on Linux and 3 targets on Windows (all simultaneously in parallel), while only paying for 2 seats. My builds are failing due to git missing, but I don't think there was an issue with the number of seats. I think the windows images may have successfully been able to replicate how the Linux images work in regard to seats.

Reply:
That's very interesting. There was no work to make HWIDs match like on Linux which sounded like it was key to prevent occupying multiple seats. Perhaps docker makes the environments similar enough now that it doesn't care since we're activating by serial?